scame/__version__.py

```diff
@@ -1,4 +1,4 @@
 """
 Keeps the version of the project.
 """
-VERSION = '0.3.1'
+VERSION = '0.3.4'
```

Comment:
I think that this should be 0.4.0 as is add a new feature... 0.3.X is for bugfixes